lms/static/js/spec/student_account/login_spec.js

```diff
@@ -267,13 +267,25 @@
             });
 
             it('displays an error if there is no internet connection', function () {
+                var clock,
+                    oldTimeout,
+                    timeout;
+
+                // We're defining "no internet connection" in this case as the
+                // request timing out.  We use a combination of the sinon fake
+                // timer and jQuery.ajaxSetup() to force a request timeout.
+                clock = sinon.useFakeTimers();
+                oldTimeout = $.ajaxSetup().timeout;
+                timeout = 1;
+                $.ajaxSetup({timeout: timeout});
+
                 createLoginView(this);
 
                 // Submit the form, with successful validation
                 submitForm(true);
 
-                // Simulate an error from the LMS servers
-                AjaxHelpers.respondWithError(requests, 0);
```

Comment:
Actually, I see that the test case above this is already testing a 500 return code.